src/api/ForForeldre.ts

```diff
@@ -0,0 +1,65 @@
+export interface TextPictureCard {
+  title: string;
+  text: string;
+  imgURL: URL;
+}
+export interface ForForeldreContent {
+  title: string;
+  ingress: string;
+  cards: Array<TextPictureCard>;
+  bottomText: string;
+}
+
+export default (): ForForeldreContent => {
+  return {
+    title: "Informasjon for foreldre",
+    ingress:
+      "Lurer du som forelder på mer om Vektorprogrammet? Her finner du informasjon om våre assistenter, kvalitetssikring, og foreldrekurs.",
+    cards: [
+      {
+        title: "Assistentene",
+        text: `Et verv som assistent hos Vektorprogrammet er 
+        ettertraktet blant studenter. Det gjør at vi kan håndplukke 
+        de aller beste til jobben. De er faglig sterke, som 
+        brenner for matematikk. I tillegg har alle assistentene 
+        våre vært gjennom et forberedelseskurs før de blir 
+        utplassert på skolene. Du kan derfor være sikker på 
+        at assistentene våre gjør en god jobb.`,
```

Comment:
Is a `\n` missing here? The two other `ForForeldreContent` has a `\n` at the end.